### PR TITLE
Issue 1 create entry

### PR DESCRIPTION
**Pull request by CollinHovey**
chrome_CbJCPbW23J
chrome_tNSwSANPu4
chrome_df13hbsAMp
chrome_MsYSFJzdTW

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+$newPhotoURL.addEventListener('input', updatePhoto);
+
+function addEntry(event) {
+  event.preventDefault();
+  var newEntry = {};
+  newEntry.entryId = data.nextEntryId;
+  newEntry.title = $title.value;
+  newEntry.photoURL = $newPhotoURL.value;
+  newEntry.notes = $notes.value;
+  data.entries.push(newEntry);
```
Comment (BrettAlbright) [inline js/main.js]: this should be `unshift` you want to add the elements to the front of the array

File: js/main.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  data.nextEntryId += 1;
+  $newPhotoURL.value = '';
+  $photoURL.setAttribute('src', 'images/placeholder-image-square.jpg');
+  $title.value = '';
```
Comment (BrettAlbright) [inline js/main.js]: its better to reset the form than to manually set the values to empty strings

File: js/main.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  $notes.value = '';
+  var dataJSON = JSON.stringify(data);
+  localStorage.setItem('allEntries', dataJSON);
```
Comment (BrettAlbright) [inline js/main.js]: local storage functionality should only be in the `data.js` file, and you should be storing the whole data object beforeunload.

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
looking good, a few updates are needed

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**